packages/nodes-base/nodes/Airtop/actions/interaction/type.operation.ts

```diff
@@ -43,7 +43,6 @@ export const description: INodeProperties[] = [
 	},
 	{
 		...elementDescriptionField,
-		placeholder: 'the search box',
 		displayOptions: {
```

Comment:
for consistency, let's add some placeholder text to the `Text` field:
<img width="481" alt="image" src="https://github.com/user-attachments/assets/e5e0eaec-6c5d-475b-8980-f6ff7c732e66" />